Code/IO/include/sitkImageFileWriter.h

```diff
@@ -27,6 +27,10 @@
 #include <memory>
 
 namespace itk {
+
+// Forward decalaration for pointer
+class ImageIOBase;
+
```

Comment:
Following sitk::ImageReaderBase, please add the following forward declaration here:


template<class T>
class SmartPointer;